cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectAssemblyOperator.java

```diff
@@ -123,6 +119,7 @@ public KafkaConnectAssemblyOperator(Vertx vertx, PlatformFeaturesAvailability pf
 
         boolean connectHasZeroReplicas = connect.getReplicas() == 0;
 
+        final String[] desiredLogging = new String[1];
```

Comment:
Unless you know that the write-site and read-site will be running on the same thread (and maybe they are, but I'm not 100% certain about that) then you should use an `AtomicReference`.